interactive/models.py

```diff
@@ -99,6 +99,7 @@ class AnalysisRequest(models.Model):
     codelist = models.CharField(max_length=255, verbose_name="Codelist")
     start_date = models.DateField()
     end_date = models.DateField()
+    commit = models.CharField(max_length=40, verbose_name="Repo commit", null=True)
```

Comment:
Would `sha` be a better name here since commit has a particular meaning in git?

Reply:
I'll got with `commit_sha`